feat(Uploader): add reupload (#11854)
youzan/vant: packages/vant/src/uploader/README.md

| disabled | Whether to disabled the upload | _boolean_ | `false` |
| readonly | Whether to make upload area readonly | _boolean_ | `false` |
| deletable | Whether to show delete icon | _boolean_ | `true` |

> | reupload | Whether to open reupload, when enabled, preview is closed | _boolean_ | `false` |

Reviewer: ```suggestion
| reupload | Whether to enable reupload, if enabled, the image preview will be disabled | _boolean_ | `false` |
```
